src/components/row/column/column.d.ts

```diff
@@ -1,36 +1,20 @@
 import * as React from 'react';
 
 export interface ColumnProps {
-    /**
-     * Classes to apply to the component.
-     */
+    /** Children. */
+    children: React.ReactNode;
+    /** Classes to apply to the component. */
     className?: string;
-
-    /**
-     * Classes applied by row component to affect all rows
-     */
+    /** Total column count. */
+    columns?: number | string;
+    /** Classes applied by row component to affect all rows */
     columnClasses?: string;
-
-    /**
-     * Alignment of content within column.
-     */
+    /** Alignment of content within column. */
     columnAlign?: 'left' | 'right';
```

Comment:
This can be any string in the propTypes but limited to "left" or "right" here, I think probably the propTypes one needs to change to reflect this

Reply:
I see that these aren't really your changes but worth doing IMO while you changing them